src/SFA.DAS.EmployerIncentives.Enums/IncentivePhase.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.EmployerIncentives.Enums
+{
+    public enum IncentivePhase : byte
```

Comment:
This should be a value object Phase with a constructor that accepts the phase string or byte (or whatever is to be stored in the db against the application or incentive)

Reply:
why?